src/components/TitleTransferPanel/Address/Address.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactElement } from "react";
+import css from "./Address.module.scss";
+
+interface AddressProps {
+  title: string;
+  name: string;
```

Comment:
```suggestion
  name?: string;
```

There will not always be a name.. some just doesn't resolve. You need to account for those as well.

Reply:
Thinking if there is a better name than `name`. Maybe `mappedIdentifier` or `identifier`, might be hard to understand what name is, and it may not be a name also, might be UEN, swift code, DNS, etc